### PR TITLE
Portfolio: mobile version

### PR DESCRIPTION
**Pull request by XsidX**
-Add projects and about section to mobile view
-Fix header background

### CODE REVIEW

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @XsidX , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, for the `mobile version` review which was supposed to be submitted on day 2 titled `setup`

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by XsidX** [comment]
Hello @Munsa1 
What do you mean, *which was supposed to be submitted on day 2 titled setup*, should I rename this pull request? Because they only have similar names but are totally different.

**Comment by bellom** [comment]
@XsidX  - Looking into it, hang in there.

CR Leader.

**Comment by XsidX** [comment]
> @XsidX - Looking into it, hang in there.
> 
> CR Leader.

Perfect...

**Comment by bellom** [comment]
@XsidX - I noticed you already have mobile-view approved, can you please use the right milestone name, and also can you tell me the name of the current milestone on your dashboard.

**Comment by XsidX** [comment]
> @XsidX - I noticed you already have mobile-view approved, can you please use the right milestone name, and also can you tell me the name of the current milestone on your dashboard.

Oh, I see
Current milestone name: [Portfolio: mobile version]

**Comment by bellom** [comment]
Yes, please.

What was the name of the previous milestone that was approved on your dashboard?

**Comment by XsidX** [comment]
> Yes, please.
> 
> What was the name of the previous milestone that was approved on your dashboard?

The previous milestone was: [Portfolio: setup and mobile version skeleton]

**Review by sinansevgi — COMMENTED** [review]
Hello, @XsidX  👋,

_This is an additional review as a result of the second opinion request._

Since the name of this milestone is `Portfolio: mobile version`, I think your PR title is well descriptive and you can use it. Since you still have review limits left, please ask for another review in your Dashboard, and the next Code Reviewer will leave a **review** for your project.

Keep rocking 🚀!

**Comment by XsidX** [comment]
> Hello, @XsidX 👋,
> 
> _This is an additional review as a result of the second opinion request._
> 
> Since the name of this milestone is `Portfolio: mobile version`, I think your PR title is well descriptive and you can use it. Since you still have review limits left, please ask for another review in your Dashboard, and the next Code Reviewer will leave a **review** for your project.
> 
> Keep rocking 🚀!

Great, thank you. I already did that.

File: styles.css [hunk 1]
```diff
@@ -56,35 +96,127 @@ ul {
...
+  height: 42%;
+  display: block;
+}
+
+.project-snapshot img {
+  width: 100%;
+  height: 100%;
+}
+
+.project-container h2 {
```
Comment (phelian23) [inline styles.css:138]: - [ ] Kindly add appropriate margins between the sections, and also between each project cards
Your designs
Capture
Capture1
Capture2
Figma Designs
figma
figma2
figma3

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @XsidX, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ Correct use of Github Flow
✅ Good commit messages
✅ Good README template

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vic778 — APPROVED** [review]
## Hi @XsidX   , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**